ops/ansible/roles/bfd-db-migrator/defaults/main.yml

```diff
@@ -5,3 +5,4 @@ db_migrator_user: bb-migrator
 db_migrator_tmp_dir: /tmp
 migrator_monitor_enabled: true
 migrator_monitor_heartbeat_interval_seconds: 300
+sqs_queue_name: bfd-test-migrator
```

Comment:
Dont the other envs need this too?

Reply:
This was actually just so the ansible tests would pass given the way we pass these environment variables. I've moved this default into `test_basic.yml` to make that a little clearer :)